modules/tinymce/src/plugins/fullscreen/main/ts/api/Events.ts

```diff
@@ -2,6 +2,7 @@ import Editor from 'tinymce/core/api/Editor';
 
 const fireFullscreenStateChanged = (editor: Editor, state: boolean): void => {
   editor.dispatch('FullscreenStateChanged', { state });
+  editor.dispatch('ResizeEditor', { state });
```

Comment:
This isn't correct, as `ResizeEditor` shouldn't have a state.

Reply:
Also if you're making this change then we should look at removing some of the duplicate event hooks we have (for example in `ControlSelection.ts`).

Reply:
There doesn't seem to be any other overlap, except for the one in the file you provided. However there's a few places where we could change events, though I'm not sure if that's what we want to do.

Reply:
Yeah we don't want to change the fullscreen event to resize editor, only remove it where it's duplicated.